tests/widget/modal/widget-modal.login-social-redirect.test.js

```diff
@@ -0,0 +1,15 @@
+import { expect, test } from '@playwright/test';
+
+test.only('Modal widget with social callback redirect', async ({ page }) => {
+  await page.goto('widget/modal?journey=LoginWithSocial');
+
+  const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
+  await loginButton.click();
+
+  await page.fill('text="User Name"', 'demouser');
+
+  await page.locator('button', { hasText: 'Continue with Google' }).click();
```

Comment:
I think we should prefer the `getByRole` api in this case.

```suggestion
  await page.getByRole('button', { name: 'Continue with Google' }).click();
```